<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buildwindow</class>
 <widget class="QWidget" name="buildwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(7, 7, 43);</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>781</width>
     <height>581</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 7, 43);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-668</y>
      <width>762</width>
      <height>1247</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QGroupBox" name="Build2">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build2Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build2Stars">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="Build3">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build4Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build3Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="Build5">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build7Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build5Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="Build1">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QPushButton" name="Build1Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build1Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="Build4">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build3Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build4Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="Build7">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(7, 7, 43);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build5Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build7Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="Build8">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build6Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build8Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="Build6">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>301</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="Build8Button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>311</width>
          <height>281</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLabel" name="Build6Stars">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>230</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(17, 17, 80, 255), stop:1 rgba(85, 170, 255, 255));</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
